src/mlpack/methods/ann/layer/convolution.hpp

```diff
@@ -74,7 +75,8 @@ class Convolution
               const size_t padW = 0,
               const size_t padH = 0,
               const size_t inputWidth = 0,
-              const size_t inputHeight = 0);
+              const size_t inputHeight = 0,
+              const std::string paddingType = "None");
```

Comment:
I'm wondering if implementing another template argument (policy) would be more consistent with the existing mlpack code, let me know what you think. I don't have a preference here.

Reply:
It looks like we cannot use `string literals` as template arguments. Should I create new classes for policies like we did for `GAN` to have `StandardGan` and `DCGAN` as class?

Reply:
Currently I am keeping it same. It looks like I will need to create classes for it as a template argument. Let me know if it's incorrect. I will change it.

Reply:
I think a string is fine, even closer to tf/keras.